pkg/plugins/policies/meshproxypatch/api/v1alpha1/meshproxypatch.go

```diff
@@ -0,0 +1,240 @@
+// +kubebuilder:object:generate=true
+package v1alpha1
+
+import (
+	common_api "github.com/kumahq/kuma/api/common/v1alpha1"
+)
+
+// MeshProxyPatch is a policy that lets you patch standard Envoy configuration
+// generated by Kuma Control Plane.
+// +kuma:policy:skip_registration=true
+type MeshProxyPatch struct {
+	// TargetRef is a reference to the resource the policy takes an effect on.
+	// The resource could be either a real store object or virtual resource
+	// defined inplace.
+	TargetRef common_api.TargetRef `json:"targetRef,omitempty"`
```

Comment:
```suggestion
	TargetRef common_api.TargetRef `json:"targetRef"`
```
We updated the policy bootstrap generator to avoid `omitempty`, but it doesn't update already generated policies